src/frequenz/sdk/actor/_data_sourcing/_component_metric_request.py

```diff
@@ -46,6 +46,7 @@ def get_channel_name(self) -> str:
             A string denoting a channel name.
         """
         return (
-            f"component-stream::{self.component_id}::{self.metric_id.name}::"
-            f"{self.start_time}::{self.namespace}"
+            f"component_metric_request<namespace={self.namespace},"
+            f"component_id={self.component_id},"
+            f"start={self.start_time}>"
```

Comment:
I wonder why we are no longer adding `self.metric_id.name`. I think it should be added but I might be missing something here

Reply:
Oh, damn, it was accidentally removed during some manipulation, thanks for catching it!!! This must be breaking the tests and I was saying "Fuck! I just wanted to do this tiny change, how could it be breaking the tests!!!" :laughing: :facepalm:

Reply:
Absolutely, this won't be unique without the metric name, and would lead to the same channels getting used for all metrics we will get a lot more than what we asked for.